Sources/VaporToolbox/Build.swift

```diff
@@ -25,107 +30,100 @@ public final class Build: Command {
     }
 
     public func run(arguments: [String]) throws {
-        if arguments.options["clean"]?.bool == true {
-            let clean = Clean(console: console)
-            try clean.run(arguments: arguments)
-        }
+        try clean(arguments)
+        try fetch(arguments)
+        try build(arguments)
+        try run(arguments)
+    }
 
-        if arguments.options["fetch"]?.bool != false {
-            let fetch = Fetch(console: console)
-            try fetch.run(arguments: [])
-        }
+    private func clean(_ arguments: [String]) throws {
+        guard arguments.flag("clean") else { return }
+        let arguments = arguments.removeFlags(["clean"])
+        let clean = Clean(console: console)
+        try clean.run(arguments: arguments)
+    }
 
-        if arguments.options["modulemap"]?.bool != false {
-            do {
-                let mod = try console.backgroundExecute(program: "/bin/sh", arguments: ["-c", "ls Packages | grep CLibreSSL"]).trim()
-                _ = try console.backgroundExecute(program: "ls", arguments: ["Packages/\(mod)/Sources/CLibreSSL/include/module.modulemap"])
-            } catch {
-                // not found
-                do {
-                    _ = try console.backgroundExecute(program: "/bin/sh", arguments: ["-c", "printf \"module CLibreSSL {\\n    header \\\"CLibreSSL.h\\\"\\n    link \\\"CLibreSSL\\\"\\n}\" > Packages/CLibreSSL-1.0.0/Sources/CLibreSSL/include/module.modulemap"])
-                } catch {
-                    console.warning("Could not add CLibreSSL Module Map: \(error)")
-                }
-            }
-        }
+    private func fetch(_ arguments: [String]) throws {
+        // unless explicitly false, treat as true
+        let shouldFetch = arguments.option("fetch")?.bool ?? true
+        guard shouldFetch else { return }
 
-        var buildFlags: [String] = []
+        let arguments = arguments.removeFlags(["clean", "fetch"])
+        let fetch = Fetch(console: console)
+        try fetch.run(arguments: arguments)
+    }
 
-        if arguments.flag("debug") {
-            buildFlags += [
-                "-Xswiftc",
-                "-g"
-            ]
-        }
+    private func build(_ arguments: [String]) throws {
+        let buildFlags = try loadBuildFlags(arguments)
 
-        buildFlags += try Config.buildFlags()
+        let isVerbose = arguments.isVerbose
```

Comment:
this same thing (59-70) gets repeated a lot in the code. can we move it to an extension?